playerbot/strategy/druid/DruidTriggers.h

```diff
@@ -166,4 +166,71 @@ namespace ai {
     public:
         NaturesSwiftnessTrigger(PlayerbotAI* ai) : BuffTrigger(ai, "nature's swiftness") {}
     };
+
+    class ForceOfNatureTrigger : public BuffTrigger
+    {
+    public:
+        ForceOfNatureTrigger(PlayerbotAI* ai) : BuffTrigger(ai, "force of nature") {}
+    };

```

Comment:
For the easier Triggers let's uses the existing Macros in the Future or create new ones, makes the File easier to read

Reply:
done